schemas/csdl.schema.json

```
@@ -248,7 +254,7 @@
         "@": {
```

```suggestion
        "^@": {
```
Sometime you use `"@"`, sometimes `"^@"`. What's the difference?

---

Good catch: the `"@"` is only needed for enumeration members whose value is not an object. Annotations on enum members are prefixed with the member name, for example `"red@Core.Description":"The color red"`

---

The `@` must be preceded by a valid identifier (and not just any valid identifier, but one that identifies a property of the complex type).